### PR TITLE
Added metadata fields under Settings tab

### PR DESCRIPTION
**Pull request by utkarshcmu**
Replacing #3873

### CODE REVIEW

**Comment by utkarshcmu** [comment]
For PR reviews and comments, please see #3873

**Comment by utkarshcmu** [comment]
Review/Mege?

File: public/app/features/dashboard/dashboardCtrl.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ function (angular, $, config, moment) {
 
         dashboardKeybindings.shortcuts($scope);
 
+        $scope.evaluateTotalStats();
```
Comment (torkelo) [inline public/app/features/dashboard/dashboardCtrl.js]: this should not be done on dashboard load, can your remove the stats from this PR completley?

**Comment by utkarshcmu** [comment]
@torkelo Removed totalStats completely from this PR. Ready for merge now. Let me know if you want any other changes. Thanks.

**Pull request merged**